pkg/pillar/cmd/agent.go

```diff
@@ -0,0 +1,63 @@
+package cmd
```

Comment:
Missed this earlier..

No.. the package should be agentbase.

This should be directly under pillar/agentbase/ directory. This should not be under cmd.

cmd -- >This directory is for actual microservices.. Any libraries should be outside of this,